### PR TITLE
Final Touches

### PR DESCRIPTION
**Pull request by jadibdev**
### DESCRIPTION
Added new styling, improved the look and feel of the app, fixed linter errors.

### DEMO
Screen Shot 2022-06-16 at 8 13 39 PM

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -12,14 +23,24 @@
...
 }
 
 #refresh {
-  height: 20px;
+  height: 40px;
   width: 75px;
+  background: #192757;
+  color: white;
+  padding-bottom: 10px;
+  padding-top: 10px;
```
Comment (ahmetbozaci) [inline src/style.css]: - [ ] To make dry css kindly combine them. `padding: 10px 0 10px 0`
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/style.css [hunk 2]
```diff
@@ -43,7 +64,13 @@ form {
...
 
 #submit {
-  height: 20px;
+  height: 40px;
   width: 75px;
   align-self: flex-end;
+  background: #192757;
+  color: white;
+  padding-bottom: 10px;
+  padding-top: 10px;
```
Comment (ahmetbozaci) [inline src/style.css]: - [ ] To make dry css kindly combine them. `padding: 10px 0 10px 0`
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

File: src/style.css [hunk 3]
```diff
@@ -12,14 +23,24 @@
...
   display: block;
   margin-bottom: 10px;
 }
 
 #refresh {
-  height: 20px;
+  height: 40px;
   width: 75px;
+  background: #192757;
+  color: white;
```
Comment (ahmetbozaci) [inline src/style.css]: - [ ] For DRY CSS you should combine duplicate properties. (color: white)
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# STATUS: Changes Required  ♻️


 @jadibdev  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Nice to have 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- Well done for following the design layout.
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._
- [ ] To make a professional readme file please fix the broken link in the license section.

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by BenMukebo — APPROVED** [review]
### Hi @jadibdev, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] Kindly, Add a descriptive README file to your project,  A professional README file should have a project title, project description, list of technology used in the project, and instructions to set and run the project up locally, This includes `git clone URL` , `cd project_name`,  `npm install`, `npm start`, etc. You can also consider adding a screenshot of the project to the README file to improve his profectionalisme😜 

- Please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

- The README file It is the first file a person will see when they encounter your project, so it should be fairly brief but detailed. It will make your project stand out from a bunch of others.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**